NCEP/gdas_utility.py

```diff
@@ -635,11 +635,11 @@ def get_dataarray(self, grbfile, var_name, level_type, desired_level):
         os.environ['AWS_CONFIG_FILE']=custom_config_file
 
     # check environment variables
-    if (download_source == 's3') & (os.environ.get('AWS_SHARED_CREDENTIALS_FILE') is None) | (os.environ.get('AWS_CONFIG_FILE') is None):
-        if os.environ.get('AWS_SHARED_CREDENTIALS_FILE') is None:
-            raise ValueError('Please set up environment varialbes AWS_SHARED_CREDENTIALS_FILE')
-        if os.environ.get('AWS_CONFIG_FILE') is None:
-            raise ValueError('Please set up environment varialbes AWS_CONFIG_FILE')
+        if (download_source == 's3') & (os.environ.get('AWS_SHARED_CREDENTIALS_FILE') is None) | (os.environ.get('AWS_CONFIG_FILE') is None):
```

Comment:
@flampouris May I ask why you need the change here?

Reply:
@junwang-noaa , @SadeghTabas-NOAA covered it. In principle, I was testing with publicly available data, e.g., NOMADS, and the conditional statement required to have AWS credentials.

Reply:
@flampouris Since Sadegh will make a PR to address the issue, so we don't need the changes here.

Reply:
@flampouris I also wanted to note that in the next PR we will update S3 credentials for public use, so you can use s3 source option. The updated s3 bucket also provide historical gdas data for graphcast ICs. Thanks